src/compiler.adoc

```diff
@@ -1125,26 +1125,36 @@ TBD
 
 ==== Integrating with CI
 
-Most continuous integration tools and services expect that test scripts you provide
-return a standard exit code. But ClojureScript by default only runs the tests and outputs
-the results to standard output. So the compiled file can not be used directly with
-continuous integration tools because it does not return any return code when something
-fails. It always return the return code `0`.
+Most continuous integration tools and services expect that test scripts you
+provide return a standard exit code. But the ClojureScript test framework
+cannot customize this exit code without some configuration, because JavaScript
+lacks a universal exit code API for ClojureScript to use.
```

Comment:
- explain: why the test framework cannot modify the exit code
- detail: add that it requires customization